### ISSUE TITLE
[Question] Why in TD3 mean q value is used instead of minimum?

### ISSUE THREAD

**Issue by yuvalgos** [DLR-RM/stable-baselines3#1112]
### Question

In the implementation of TD3, in the train method, i noticed that both when computing the target Q values and the actor loss, the value is a computed as the mean of all critics output:
```            
# Get current Q-values estimates for each critic network
current_q_values = self.critic(replay_data.observations, replay_data.actions)

# Compute critic loss
critic_loss = sum([F.mse_loss(current_q, target_q_values) for current_q in current_q_values])
```
...
```
actor_loss = -self.critic.q1_forward(replay_data.observations, self.actor(replay_data.observations)).mean()
```
While the smallest value has to be taken instead, to reduce overestimation.
This is mentioned both in the papper:
![image](https://user-images.githubusercontent.com/2778966/195389863-38bbc787-5952-4416-a625-b7e438e80f0d.png)
and in the OpenAI Spinning guide mentioned in the documentation:
https://spinningup.openai.com/en/latest/algorithms/td3.html#key-equations

Is there a particular reason why in sb3 the mean is used? 

### Additional context
The relevant code:
https://github.com/DLR-RM/stable-baselines3/blob/508f8ffd598482701813c249e397547f7d91e1c1/stable_baselines3/td3/td3.py#L145

**Comment by yuvalgos** [comment]
Oh I totally missed it. Now i see it thank you!
